deployment/base/ftl-controller/role.yaml

```diff
@@ -20,3 +20,6 @@ rules:
     resourceNames:
       - ftl-controller-deployment-config
     verbs: [ "get"]
+  - apiGroups: [ "security.istio.io" ]
+    resources: [ "authorizationpolicies" ]
+    verbs: [ "get", "list", "watch", "delete", "create", "update", "patch" ]
```

Comment:
This will get moved to the provisioner at some point. 

For now I could probably drop down to just the 'create' role, using owner references for deletion, but that will likely change if we start doing P2P and need to modify them as deployments come online.

Reply:
https://github.com/TBD54566975/ftl/pull/2936/files reduces the permissions somewhat